PO_TESTING.md

```diff
@@ -0,0 +1,198 @@
+# Test PO File Translation Feature
```

Comment:
Also can you not write lengthy markdown files for no good reason. I mean the readme is already super long. No need to write essays about this. just delete it. It contains no value.